tests/circuit_graph/test_circuit_graph.py

```diff
@@ -348,12 +376,24 @@ def test_print_contents(self):
 
         f = io.StringIO()
         with contextlib.redirect_stdout(f):
-            circuit_w_wires.print_contents()
+            print(circuit_w_wires)
         out = f.getvalue().strip()
 
         expected = """Operations\n==========\nH(0)\nCNOT(wires=[0, 1])\n\nObservables\n===========\nsample(wires=[0, 1, 2])"""
         assert out == expected
```

Comment:
Now we should be able to just do `str(circuit_w_wires) == expected` and not need to do any of that string redirection stuff.

Reply:
Ok right, I'll change that!